domain-models-runtime/src/test/java/org/folio/rest/persist/ddlgen/SchemaMakerTest.java

```diff
@@ -47,6 +47,29 @@ public void canCreateAuditedTable() throws IOException, TemplateException {
                              containsString("var1 TEXT;"), containsString("var2 TEXT;"), containsString("var3 TEXT;")));
   }
 
+  @Test
+  public void canCreateCompoundIndex() throws IOException, TemplateException {
+    SchemaMaker schemaMaker = new SchemaMaker("harvard", "circ", TenantOperation.UPDATE,
+      "mod-foo-18.2.3", "mod-foo-18.2.4");
+    String json = ResourceUtil.asString("templates/db_scripts/compoundIndexTest.json");
+    schemaMaker.setSchema(ObjectMapperTool.getMapper().readValue(json, Schema.class));
+    String result = schemaMaker.generateDDL();
+
+    assertThat(result, containsString("concat_space_sql(tablea.jsonb->>'firstName' , tablea.jsonb->>'lastName')"));
+
+  }
+
+  @Test
+  public void canCreateSQLExpressionIndex() throws IOException, TemplateException {
+    SchemaMaker schemaMaker = new SchemaMaker("harvard", "circ", TenantOperation.UPDATE,
+      "mod-foo-18.2.3", "mod-foo-18.2.4");
+    String json = ResourceUtil.asString("templates/db_scripts/compoundIndexTest.json");
+    schemaMaker.setSchema(ObjectMapperTool.getMapper().readValue(json, Schema.class));
+    String result = schemaMaker.generateDDL();
+
+    assertThat(result, containsString("lower(concat_space_sql(jsonb->>'field1', jsonb->>'field2'))"));
```

Comment:
Can you also check that schemaMaker.getSchema().getTables().get(*).get*Index().get(*).createCompoundIndex("tablea") is correct?